eng/tox/create_package_and_install.py

```diff
@@ -30,51 +30,71 @@ def cleanup_build_artifacts(build_folder):
         shutil.rmtree(results[0])
 
     # clean up build results
-    shutil.rmtree(os.path.join(build_folder, "build"))
+    build_path = os.path.join(build_folder, "build")
+    if os.path.exists(build_path):
+        shutil.rmtree(build_path)
 
 
-def discover_whls(setuppy_path, args):
-    wheels = []
+def discover_packages(setuppy_path, args):
+    packages = []
     if os.getenv("PREBUILT_WHEEL_DIR") is not None and not args.force_create:
-        wheels = discover_prebuilt_whl(os.getenv("PREBUILT_WHEEL_DIR"), setuppy_path)
+        packages = discover_prebuilt_package(os.getenv("PREBUILT_WHEEL_DIR"), setuppy_path, args.package_type)
     else:
-        wheels = build_and_discover_whl(setuppy_path, args.distribution_directory, args.target_setup)
-    return wheels
+        packages = build_and_discover_package(setuppy_path, args.distribution_directory, args.target_setup, args.package_type)
+    return packages
 
 
-def discover_prebuilt_whl(whl_directory, setuppy_path):
```

Comment:
nit: next time you make a pass through this. `setup_py_path` is easier to read than `setuppy`. It's not worth resetting the checks on this one though.

Reply:
Sure. I will make a note of it. Thanks